sci-libs/geographiclib/geographiclib-1.49.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit versionator
+inherit cmake-utils
+
+PYTHON_COMPAT=( python{2_7,3_4} )
+
+inherit python-r1
+
+MY_PN="GeographicLib"
+MY_PV="$(replace_version_separator 2 '-')"
+MY_P="${MY_PN}-${MY_PV}"
+
+DESCRIPTION=""
+HOMEPAGE="https://sourceforge.net/projects/geographiclib/"
+SRC_URI="https://sourceforge.net/projects/geographiclib/files/distrib/${MY_P}.tar.gz/download -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
```

Comment:
Could you also add `~arm ~arm64 ~x86`?

Reply:
Sure, but I haven't tested this on those systems.